consensus/metrics.go

```diff
@@ -0,0 +1,24 @@
+package consensus
+
+import (
+	"github.com/go-kit/kit/metrics"
+	prmkit "github.com/go-kit/kit/metrics/prometheus"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	Namespace = "spacemesh"
+	Subsystem = "consensus"
+)
+
+func getGauge(name, help string, labels []string) metrics.Gauge {
```

Comment:
consider renaming to newGauge. 
I might take it out to a higher package and let everyone use it.